contrib/golang/filters/http/source/go/pkg/http/shim.go

```diff
@@ -351,6 +352,10 @@ func envoyGoFilterOnHttpLog(r *C.httpRequest, logType uint64,
 //export envoyGoFilterOnHttpStreamComplete
 func envoyGoFilterOnHttpStreamComplete(r *C.httpRequest) {
 	req := getRequest(r)
+	if req == nil {
```

Comment:
Should similar handling also be done in envoyGoFilterOnHttpDestroy?

Reply:
I submit a new commit to follow the DRY rule.